main/modes/system/intro/introMode.c

```diff
@@ -371,6 +395,11 @@ static void introEnterMode(void)
     loadFont("retro_logo.font", &iv->logoFont, true);
     makeOutlineFont(&iv->logoFont, &iv->logoFontOutline, true);
 
+#ifdef CUSTOM_INTRO_SOUND
+    iv->samplePlayer.sample = cnfsGetFile("magfest.bin", &iv->samplePlayer.sampleCount);
```

Comment:
Compression is for nerds! It's only 18kb anyway and a test zip (better than heatshrink) only drops it to ~14kb.

Reply:
Oh nice, I didn't check that but I also figured it might actually take a non-negligible amount of time to decompress and copy everything over on startup anyway.